seeder/src/renderer/main.css

```diff
@@ -27,4 +27,5 @@
 * {
 	outline: none !important;
 	box-sizing: border-box;
+  user-select: none;
 }
```

Comment:
we are using tailwind right?

Reply:
we are? lemme see... :confused:

Reply:
Kindly remove all these from here and use tailwind class

Reply:
Resolved on https://github.com/coreproject-moe/CoreProject/pull/3929/commits/d47db09259a7c83064d00cb08b7337bc87afedf2